content/blog/reprex-1-0-0/index.Rmd

```diff
@@ -0,0 +1,171 @@
+date: 2021-01-27
+author: Jenny Bryan
+description: >
+    We've never blogged about reprex before, so the release of v1.0.0 seems
+    like a good reason occasion for it.
+
+photo:
+  url: https://www.flickr.com/photos/iamagenious/6251271390/sizes/o/
+  author: Eric Molina
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [package] 
+tags: [reprex,tidyverse]
+---
+
+We're exhilarated to announce the release of reprex 1.0.0 ([reprex.tidyverse.org](https://reprex.tidyverse.org)).
+reprex is a package that helps you prepare **REPR**oducible **EX**amples to share in places where people talk about code, e.g., on GitHub, on Stack Overflow, and in Slack or email messages.
+
+You can install the current version of reprex from CRAN with[^1]:
+
+[^1]: Another way you might get reprex is by installing the tidyverse meta-package.
+    reprex is one of the packages installed by `install.packages("tidyverse")`, however it is **not** among the core packages attached by `library(tidyverse)`.
+
+```{r, eval = FALSE}
+install.packages("reprex")
+```
+
+It turns out we've never blogged about reprex here on [tidyverse.org](https://www.tidyverse.org), so we start with a general overview for newcomers, then close with a summary of recent changes of interest to existing users.
+
+You can see a full list of changes in the [release notes](https://reprex.tidyverse.org/news/index.html#reprex-1-0-0-2021-01-27).
+
+## Why reprex exists
+
+reprex is a convenience package that combines the power of R Markdown with conventional wisdom about what makes a good reproducible example.
+We believe that conversations about code are more productive with:
+
+-   Code that **actually runs**
+-   Code that the reader **doesn't necessarily have to run**
+-   Code that the reader **can easily run**
+
+Someone new to R might be puzzled by this code and want to discuss it:
+
+```{r}
+x <- factor("Hello, ")
+y <- factor("world!")
+x
+y
+c(x, y)
+```
+
+You, as a reader, just benefited from a few things:
+
+1.  The code was actually executed by the R interpreter, because this post is generated from an `.Rmd` document.
+    I am probably not wasting your time with fictional or incomplete code.
+
+2.  You got to see the actual result without firing up R yourself and executing this code locally.
+    Many R veterans will instantly recognize what's going on and be able to provide some useful explanation at this point.
+
+3.  If you do want to execute the code yourself, you can easily copy the entire chunk, paste it into R, and press enter[^2].
+    Compare that with the fussy edits you'd have to make with this copy/paste from the R Console:
+
+    ``` r
+    > x <- factor("Hello, ")
+    > y <- factor("world!")
+    > x
+    [1] Hello, 
+    Levels: Hello, 
+    > y
+    [1] world!
+    Levels: world!
+    > c(x, y)
+    [1] 1 1
+    ```
+
+    You need to remove the `>` prompts and delete all output.
+    And don't get me started on the problem of screenshots.
```

Comment:
Let alone the problem of photos of screens!

Reply:
Oh I see you mention that below

Reply:
Saving the best for last! 📸